packages/ckeditor5-source-editing/src/sourceediting.js

```diff
@@ -198,7 +207,7 @@ export default class SourceEditing extends Plugin {
 		// It is not needed to iterate through all editing roots, as currently the plugin supports only the Classic Editor with a single
 		// main root, but this code may help understand and use this feature in external integrations.
 		for ( const [ rootName, domRootElement ] of editingView.domRoots ) {
-			const data = editor.data.get( { rootName } );
+			const data = formatSource( editor.data.get( { rootName } ) );
```

Comment:
I was thinking if the formatter should be configurable but we agreed with @psmyrek that this configurability can be added in the future.